Extractor/Nodes/BaseUANode.cs

```diff
@@ -562,6 +562,36 @@ protected Dictionary<string, string> BuildMetadataBase(Dictionary<string, string
             return result;
         }
 
+        protected Dictionary<string, object> BuildMetadataJsonBase(Dictionary<string, object>? extras, IUAClientAccess client, bool reversibleJson = true)
```

Comment:
I really would prefer we avoided the `object` here. Could we not use `JsonNode` instead? I think that will be much easier to use with legacy metadata as well.